.github/labeler.yml

```diff
@@ -21,13 +21,15 @@ universe-public-cloud:
   - packages/manager/apps/public-cloud/**
 universe-bare-metal-cloud:
   - packages/manager/apps/dedicated/**/!(dedicatedCloud)/**
+  - packages/manager/*/billing/**
   - packages/manager/*/cda/**
   - packages/manager/*/cloud-connect/**
   - packages/manager/*/dbaas-logs/**
   - packages/manager/*/enterprise-cloud-database/**
   - packages/manager/*/iplb/**
   - packages/manager/*/metrics/**
   - packages/manager/*/nasha/**
+  - packages/manager/*/netapp/**
```

Comment:
Following location doesn't exist yet on `master` branch. Is it required?

Reply:
If I'm correct configuration is only taken into account if done on `master` branch so any PR related to this topic won't get the labels if not added